### DISCUSSION TITLE
Strange issue with audio level randomly varying [EDIT: Koord feature!]

### DISCUSSION THREAD

**Discussion by marktheharp** [jamulussoftware/jamulus#3210] [Technical Support & Help] [answered]
Recently I’ve noticed during a session and afterwards on the server recordings that the audio level I’m sending occasionally and seemingly randomly turns down about 2-3dB for quite a few seconds then gradually back up to normal again. Being careful with levels and looking ok on the desk / Jamulus meters and the interface. I use this same kit for zoom teaching and recording and don’t have problems like that anywhere else. There is no compressor anywhere but it sounds like a compressor with an 1-2 second attack / recovery.

Just wondered if anyone else has had this issue.

Doing Jamulus sessions using Koord server. Have also asked them about this in case it’s a Koord thing.

It’s possible it’s me! Will try something a bit more scientific with sinewaves of different levels to see if I can replicate the issue in a more controlled way.

But if this is anything you recognise let me know.

win 11 / soundcraft mixer / Behringer umc204hd / Jamulus v3.10.0

**Comment by dcorson-ticino-com** [comment]
Jamulus does not touch the level you send at.  
That is determined by your audio HW and the operating system drivers.
With Jamulus you can only change your level in the mix, i.e. the level you receive.
I have no good idea where this could be coming from, but would first look at the audio HW, cabling, connectors.

**Reply by marktheharp** [reply]
Don, thank you so much for the reply. I suspected as much – that Jamulus doesn’t touch anything. I know this there is an anti-feedback setting, but presumably that is just for the start of the session.

I have been in touch with Koord and they have told me they do in fact do some level correction on their completed mixes to adjust very low levels. I don’t ever use the completed mix feature as I want to download the stems to mix them myself, but I’m going to do some experiments to see if Koord is also somehow applying this to the stems – causing the effect I was hearing. From your reply, it sounds like Jamulus wouldn’t be causing this.

Thank you so much, and I will update this when I have done a bit of experimentation.

**Reply by marktheharp** [reply, marked as answer]
Just to update: it seems that when Koord processes the mix, they sometimes have to stitch together files, particularly whether has been a bit of interruption in the network.

I have now heard back a bit more from Koord about and it appears that thanks to me sending them a part of the session where this level change happens, it has identified an issue where network interruption can cause instability and make the process change the levels. They are going to update their processing of recorded files, and it shouldn’t be an issue in future. Thank you, Don, for helping me pin down exactly where in the process it was happening.
